### PR TITLE
tcld Assembly config

### PR DESCRIPTION
**Pull request by flossypurse**

### CODE REVIEW

**Review by DJSanti — CHANGES_REQUESTED** [review]
Just had one thing to say concerning a link. Other than that, everything is good to me. Probably not a bad idea to get an SME or Dail on here as well

**Comment by djmagee** [comment]
My review is still in progress.

File: docs-src/cloud/tcld/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+:::
+
+:::note Join the Temporal Cloud waitlist
+
+Access to Temporal Cloud is currently by invitation only.
+You can [join the waitlist](https://pages.temporal.io/cloud-early-access).
+
+:::
+
```
Comment (djmagee) [inline docs-src/cloud/tcld/index.md:23]:
```suggestion
```
We can make these admonitions go away entirely.

**Review by djmagee — APPROVED** [review]
Just two tweaks from me.

**Review by DJSanti — APPROVED** [review]

**Pull request merged**